ceph.spec.in

```diff
@@ -1501,7 +1501,7 @@ mkdir -p %{buildroot}%{_localstatedir}/lib/ceph/bootstrap-rbd-mirror
 install -m 644 -D monitoring/ceph-mixin/prometheus_alerts.yml %{buildroot}/etc/prometheus/ceph/ceph_default_alerts.yml
 
 # SNMP MIB
-install -m 644 -D -p monitoring/snmp/CEPH-MIB.txt %{buildroot}%{_datadir}/snmp/mibs
+install -m 644 -D -p monitoring/snmp/CEPH-MIB.txt -t %{buildroot}%{_datadir}/snmp/mibs
```

Comment:
- I suppose the intent behind adding `-t` was to ensure that `-D` would create all components, including the last (i.e. `mibs`).  In that case, should we follow the `install [OPTION]... -t DIRECTORY SOURCE...` syntax from the man page to make it really stand out?
- I don't see `-p` being specified for any other `install` invocations in our spec file.  If there is a specific reason to preserve `CEPH-MIB.txt` timestamp, let's add a comment.  If not, I'd drop `-p`. 
```suggestion
install -m 644 -D -t %{buildroot}%{_datadir}/snmp/mibs monitoring/snmp/CEPH-MIB.txt
```

Reply:
> * I suppose the intent behind adding `-t` was to ensure that `-D` would create all components, including the last (i.e. `mibs`).  In that case, should we follow the `install [OPTION]... -t DIRECTORY SOURCE...` syntax from the man page to make it really stand out?

yes.

> * I don't see `-p` being specified for any other `install` invocations in our spec file.  If there is a specific reason to preserve `CEPH-MIB.txt` timestamp, let's add a comment.  If not, I'd drop `-p`.

neither do i. dropped.